UI Test Runner/UITests/FormViewTests.swift

```diff
@@ -775,6 +775,113 @@ final class FormViewTests: XCTestCase {
             "String 1"
         )
     }
+    
+    // - MARK: Test case 5: Switch input type
+    
+    /// Test case 5.1: Test switch on
+    func testCase_5_1() {
+        let app = XCUIApplication()
+        let fieldTitle = app.staticTexts["switch integer"]
+        let formTitle = app.staticTexts["mainobservation_ExportFeatures"]
+        let formViewTestsButton = app.buttons["FormView Tests"]
+        let `switch` = app.switches["switch integer Switch"]
```

Comment:
Maybe `switchView`??
```suggestion
        let switchView = app.switches["switch integer Switch"]
```